### ISSUE TITLE
Autowiring fails when static_info_tables not installed

### ISSUE THREAD

**Issue by TheNaderio** [in2code-de/femanager#256]
```
[ Symfony\Component\DependencyInjection\Exception\RuntimeException ]         
Cannot autowire service "In2code\Femanager\DataProvider\CountryZonesDataPro  
vider": argument "$countryZoneRepository" of method "__construct()" has typ  
e "SJBR\StaticInfoTables\Domain\Repository\CountryZoneRepository" but this   
class was not found.
```

The CountryZoneProvider and CountryDataProvider uses static_info_tables, which are required only in `require-dev`.
Either the classes should be remover from `Services.yaml` or `static_info_tables` should be required by femanager.

**Comment by TheNaderio** [comment]
`ShouldValidateStateCondition` throws this exception too.

**Comment by andre-spindler** [comment]
We have the same issue here.
As soon as femanager version 6.1.0 is installed/updated, the installation breaks.
You cannot install StaticInfoTables afterwarrds to repair this, as it is not possible to create a valid PackageStates file.
What about moving the country and zone features to an additional extension and define explicit requirements there?

**Comment by dahaupt** [comment]
Just ran into this issue. It is already fixed by #287 and #290 in development branch. Looking forward to a hotfix release in order to prevent other people having to debug this.

**Comment by spoonerWeb** [comment]
If fixed already we can close this?
